flang/test/Lower/OpenACC/acc-declare.f90

```diff
@@ -401,14 +396,13 @@ module acc_declare_allocatable_test2
   subroutine init()
     use acc_declare_allocatable_test
     allocate(data1(100))
-! CHECK: fir.store %{{.*}} to %{{.*}} {acc.declare_action = #acc.declare_action<postAlloc = @_QMacc_declare_allocatable_testEdata1_acc_declare_update_desc_post_alloc>} : !fir.ref<!fir.box<!fir.heap<!fir.array<?xi32>>>>
+! CHECK: fir.call @_FortranAAllocatableAllocate
```

Comment:
This test is checking for the declare action attached to the allocate operation so changing it this way makes no sense.